### PR TITLE
Add generic `window` icons

### PR DESCRIPTION
**Pull request by danielbayley**
Related to #1003…

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
Hm, this looks a bit more like what `panel-top` should be.

This are two `window` icons from my drafts, I'd envision something more like this for an application window:
image

**Comment by danielbayley** [comment]
> Hm, this looks a bit more like what `panel-top` should be.

Yeah I was just thinking to close and add to that instead…

> This are two `window` icons from my drafts, I'd envision something more like this for an application window

But maybe keep open for app `window` vs `panel`…?

File: icons/window-mac.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "mac",
+    "macos",
+    "osx",
+    "application"
+  ],
+  "categories": [
+    "design",
```
Comment (karsa-mistmere) [inline icons/window-mac.json:10]:
```suggestion
    "design",
    "files",
```

File: icons/window.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "windows",
+    "application"
+  ],
+  "categories": [
+    "design",
```
Comment (karsa-mistmere) [inline icons/window.json:8]:
```suggestion
    "design",
    "files",
```

File: icons/window-mac.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "mac",
+    "macos",
+    "osx",
+    "application"
```
Comment (karsa-mistmere) [inline icons/window-mac.json:7]:
```suggestion
    "application",
    "executable"
```

File: icons/window.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "windows",
+    "application"
```
Comment (karsa-mistmere) [inline icons/window.json:5]:
```suggestion
    "application",
    "executable"
```

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]
Both of these are currently off center, but I'm not sure about them anyways.

I know they're essentially my designs but there's a reason I haven't submitted them yet, I would love to see someone's fresh take on this.

**Comment by ericfennis** [comment]
Are two variants not too specific? Can't we just design one for the same use case? 
Maybe it is just me but the Mac variant is not really recognizable as an application window.

**Comment by danielbayley** [comment]
> Maybe it is just me but the Mac variant is not really recognizable as an application window.

I mean to a mac user (speaking as a longterm one, exclusively) it’s pretty much iconic as an app window…

**Comment by ericfennis** [comment]
@karsa-mistmere Nice, yeah think this will do for both Mac and Windows users. Btw I am a Mac user as well.

File: icons/app-window.svg [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="20" height="14" x="2" y="5" rx="2" />
+  <line x1="2" x2="22" y1="9" y2="9" />
+  <line x1="6" x2="6" y1="5" y2="9" />
+  <line x1="10" x2="10" y1="5" y2="9" />
```
Comment (karsa-mistmere) [inline icons/app-window.svg:15]: Let's make this a bit taller, like the Github icon is.
```suggestion
  <rect x="2" y="4" width="20" height="16" rx="2" />
  <path d="M10 4v4" />
  <path d="M2 8h20" />
  <path d="M6 4v4" />
```

**Review by jguddas — APPROVED** [review]

**Pull request merged**